runtime/devnet/src/extensions/mod.rs

```diff
@@ -673,501 +495,3 @@ mod tests {
 		});
 	}
 }
-// use enumflags2::BitFlags;
```

Comment:
enumflags2 is not being used anywhere but remains in the Cargo.toml

Reply:
Thank you